src/Resources/translations/messages.fr.yaml

```diff
@@ -0,0 +1,30 @@
+setono_sylius_terms:
+  ui:
+    terms: Conditions
+    manage_terms: Gérer les conditions
+    new_terms: Nouvelles conditions
+    edit_terms: Modifier les conditions
+
+    code: Code
+    name: Nom
+    channels: Canaux
+
+  menu:
+    admin:
+      main:
+        configuration:
+          terms: Conditions
+
+  form:
+    terms_accept:
+      label: Accepter toutes les conditions pour finaliser la commande
+      terms_should_be_accepted: "{{ name }} doit être accepté "
```

Comment:
Depending on the template I think it could be more efficiant to add "pour finaliser la commande" at the end of this sentence.